dask/bag/text.py

```diff
@@ -57,10 +57,12 @@ def read_text(path, blocksize=None, compression='infer',
     --------
     from_sequence: Build bag from Python sequence
     """
+    storage_options = storage_options or {}
     if isinstance(path, (tuple, list, set)):
         blocks = sum([read_text(fn, blocksize=blocksize,
                       compression=compression, encoding=encoding, errors=errors,
-                      linedelimiter=linedelimiter, collection=False, **kwargs)
+                      linedelimiter=linedelimiter, collection=False,
+                      **storage_options)
```

Comment:
Look, you have `storage_options` argument, but here you expand its values. It should be just `storage_options=storage_options`.